Task5_2/Catalog.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Task5_2
+{
+    internal class Catalog
+    {
+        private Dictionary<string, Book> _catalog = new Dictionary<string, Book>();
+
```

Comment:
Check isbn for validity-> 2	The 13-digit ISBN of a book is a string in the format XXX-X-XX-XXXXXX-X or XXXXXXXXXXXXX, where X is the number 0..9.